### ISSUE TITLE
Create admin tool bar focus state on staging environment

### ISSUE THREAD

**Issue by EWashb** [department-of-veterans-affairs/va.gov-cms#9914]
## Background

On staging, the admin toolbar has no visible focus indicator on the elements within the toolbar. 

Image of defective area:

![Screen Shot 2022-07-27 at 12.04.19 PM.png](https://images.zenhubusercontent.com/614b6cbe05afafb268be4c0b/b9ddbfba-5cb4-4914-b9eb-fd4e5406eaad)

## Accessibility Standard

WCAG version 2.0 A, [Criterion 2.4.3](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html#:~:text=3%3A%20Focus%20Order-,Success%20Criterion%202.4.,that%20preserves%20meaning%20and%20operability.)

## Acceptance Criteria

- [x] Determine if this is an upstream issue
- [x] When elements in the toolbar receive focus, a visible focus indicator should be shown
- [x] tech feasibility review 
- [x] If design is needed, separate ticket should be created for the UX (no ticket required)
- [ ] change management is consulted
- [ ] Use the current tool bar hover state and add a standard green outline to create a focus state for the tool bar.

**Comment by EWashb** [comment]
Will need to determine if these are Core or the module

**Comment by EWashb** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/VAgov-CMS-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/9914?planning-poker) with ZenHub @cjterdi

**Comment by cjterdi** [comment]
Not sure if I should be contributing to the time estimate on this ticket. If design is needed, it will be a light lift.

**Comment by EWashb** [comment]
@cjterdi Don't worry about the estimate yet. This was when I was attempting to use planning poker. I don't think that will be the best tool for tickets like this after all.

**Comment by EWashb** [comment]
@cjterdi this is not a priority for sprint 66, however, it is refined and can be picked up if capacity allows

**Comment by cjterdi** [comment]
Use the current tool bar hover state and add a standard green outline to create a focus state for the tool bar. No additional design is required.

**Comment by laflannery** [comment]
@edmund-dunn I remember talking about this one now - It is ready to go

**Comment by EWashb** [comment]
@edmund-dunn was this ever completed? I just want to make sure before I close it or move it.

**Comment by laflannery** [comment]
@EWashb This fell off because it didn't have the correct label so I lost track of it.  It has not been completed so I am adding the necessary labels and putting it back in the backlog

**Comment by laflannery** [comment]
Closing this in favor of #12549
